src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/environment_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,169 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing the environment in which Meltano is operating",
```

Comment:
Total nit, but a slight tweak here makes it more clear we aren't talking about the 'environments' feature.

```suggestion
    "description": "Schema for a Snowplow context describing the environment in which the Meltano process is executing",
```